docs/pages/access-controls/guides/ip-pinning.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: "IP Pinning (Preview)"
```

Comment:
I'm not sure if a _guide_ is the right format for this page.

Guides make sense when you need to outline a step-by-step process. There aren't a lot of steps for IP pinning, you just enable it in a role and it works transparently in the background.

See https://goteleport.com/docs/contributing/documentation/style-guide/#kinds-of-pages for the kinds of pages we have. I feel like a conceptual guide or reference page would be better for this type of content.

Reply:
I've modeled new page after `per-session MFA` page a bit, since it's kinda similar in scope and it's also not really a guide. All pages in `Cluster Access and RBAC` are located in `guides` folder, including per-session MFA, maybe we should change it. I'll check on it with docs team 👍